client/lib/products-values/index.js

```diff
@@ -217,6 +217,13 @@ function isDomainMapping( product ) {
 	return product.product_slug === 'domain_map';
 }
 
+function isDomainMappingNotMappedToWordPressDotCom( product ) {
```

Comment:
I'm a bit confused, the name of the function suggests the domain is *not* mapped to WordPress.com, but
what is checked that it is actually mapped:

```js
return product.product_slug === 'domain_map' && product.mappedDomainHasWpcomNameServers
```

What am I missing?